Add new ERT mode added

```
FAILED tests/test_units/test_fmuprovider_class.py::test_ert_simulation_modes_one_to_one - AssertionError: assert {'ensemble_smoother', 'workflow', 'ensemble_information_filter', 'evaluate_ensemble', 'test_run', 'ensemble_experiment', 'es_mda', 'manual_update'} == {'ensemble_smoother', 'workflow', 'evaluate_ensemble', 'test_run', 'ensemble_experiment', 'es_mda', 'manual_update'}
  
  Extra items in the left set:
  'ensemble_information_filter'
  
  Full diff:
    {
        'ensemble_experiment',
  +     'ensemble_information_filter',
        'ensemble_smoother',
        'es_mda',
        'evaluate_ensemble',
        'manual_update',
        'test_run',
        'workflow',
    }
```

ENiF has been added. This should be added to ERT enums.

--- a/.gitignore
+++ b/.gitignore
@@ -108,3 +108,6 @@ src/fmu/dataio/version.py
 node_modules/
 package-lock.json
 package.json
+
+# manifest file in examples
+examples/example_exports/.dataio_export_manifest.json

--- a/examples/example_metadata/fmu_case.yml
+++ b/examples/example_metadata/fmu_case.yml
@@ -32,7 +32,7 @@ masterdata:
       uuid: ad214d85-8a1d-19da-e053-c918a4889310
 source: fmu
 tracklog:
-- datetime: '2025-05-23T10:18:09.423131Z'
+- datetime: '2025-06-11T07:37:46.256673Z'
   event: created
   sysinfo:
     fmu-dataio:

--- a/examples/example_metadata/polygons_field_outline.yml
+++ b/examples/example_metadata/polygons_field_outline.yml
@@ -63,7 +63,7 @@ fmu:
     name: iter-0
     uuid: 00000000-0000-0000-0000-000000000000
   entity:
-    uuid: 6f62ba89-c3d7-d6d8-f6ee-f6237aa105ca
+    uuid: 85a58443-3beb-23f8-c91a-86ff6e247346
   ert:
     experiment:
       id: 00000000-0000-0000-0000-000000000000
@@ -100,7 +100,7 @@ masterdata:
       uuid: ad214d85-8a1d-19da-e053-c918a4889310
 source: fmu
 tracklog:
-- datetime: '2025-05-23T10:18:12.581493Z'
+- datetime: '2025-06-11T07:37:51.865324Z'
   event: created
   sysinfo:
     fmu-dataio:

--- a/examples/example_metadata/polygons_field_region.yml
+++ b/examples/example_metadata/polygons_field_region.yml
@@ -63,7 +63,7 @@ fmu:
     name: iter-0
     uuid: 00000000-0000-0000-0000-000000000000
   entity:
-    uuid: f78ac9ce-1f52-f473-e32a-244b1d097965
+    uuid: cdd06e5d-e104-ebbd-1c45-5e25b914522f
   ert:
     experiment:
       id: 00000000-0000-0000-0000-000000000000
@@ -100,7 +100,7 @@ masterdata:
       uuid: ad214d85-8a1d-19da-e053-c918a4889310
 source: fmu
 tracklog:
-- datetime: '2025-05-23T10:18:12.556357Z'
+- datetime: '2025-06-11T07:37:51.831363Z'
   event: created
   sysinfo:
     fmu-dataio:

--- a/examples/example_metadata/preprocessed_surface_depth.yml
+++ b/examples/example_metadata/preprocessed_surface_depth.yml
@@ -80,7 +80,7 @@ masterdata:
       uuid: ad214d85-8a1d-19da-e053-c918a4889310
 source: fmu
 tracklog:
-- datetime: '2025-05-23T10:18:17.059617Z'
+- datetime: '2025-06-11T07:38:01.668196Z'
   event: created
   sysinfo:
     fmu-dataio:

--- a/examples/example_metadata/surface_depth.yml
+++ b/examples/example_metadata/surface_depth.yml
@@ -60,7 +60,7 @@ fmu:
     name: iter-0
     uuid: 00000000-0000-0000-0000-000000000000
   entity:
-    uuid: 92c2dfac-e9a4-43c1-7598-b3f569f05bdd
+    uuid: 3135c777-e85a-0f80-63d3-837831b31e8a
   ert:
     experiment:
       id: 00000000-0000-0000-0000-000000000000
@@ -97,7 +97,7 @@ masterdata:
       uuid: ad214d85-8a1d-19da-e053-c918a4889310
 source: fmu
 tracklog:
-- datetime: '2025-05-23T10:18:18.589394Z'
+- datetime: '2025-06-11T07:38:04.314494Z'
   event: created
   sysinfo:
     fmu-dataio:

--- a/examples/example_metadata/surface_fluid_contact.yml
+++ b/examples/example_metadata/surface_fluid_contact.yml
@@ -63,7 +63,7 @@ fmu:
     name: iter-0
     uuid: 00000000-0000-0000-0000-000000000000
   entity:
-    uuid: cf8064ba-8a5f-808c-d0ff-4d5bd4ae1549
+    uuid: 4a055a3a-470f-80ee-ed18-9e3c4594f604
   ert:
     experiment:
       id: 00000000-0000-0000-0000-000000000000
@@ -100,7 +100,7 @@ masterdata:
       uuid: ad214d85-8a1d-19da-e053-c918a4889310
 source: fmu
 tracklog:
-- datetime: '2025-05-23T10:18:18.642781Z'
+- datetime: '2025-06-11T07:38:04.383548Z'
   event: created
   sysinfo:
     fmu-dataio:

--- a/examples/example_metadata/surface_seismic_amplitude.yml
+++ b/examples/example_metadata/surface_seismic_amplitude.yml
@@ -81,7 +81,7 @@ fmu:
     name: iter-0
     uuid: 00000000-0000-0000-0000-000000000000
   entity:
-    uuid: 8059d8b0-933d-f9af-1b2c-5c7dffeb5339
+    uuid: 699b770d-bae9-d463-f1f7-a1c7c6f18162
   ert:
     experiment:
       id: 00000000-0000-0000-0000-000000000000
@@ -118,7 +118,7 @@ masterdata:
       uuid: ad214d85-8a1d-19da-e053-c918a4889310
 source: fmu
 tracklog:
-- datetime: '2025-05-23T10:18:18.694033Z'
+- datetime: '2025-06-11T07:38:04.457370Z'
   event: created
   sysinfo:
     fmu-dataio:

--- a/examples/example_metadata/table_inplace_volumes.yml
+++ b/examples/example_metadata/table_inplace_volumes.yml
@@ -68,7 +68,7 @@ fmu:
     name: iter-0
     uuid: 00000000-0000-0000-0000-000000000000
   entity:
-    uuid: 301eb71e-252c-c809-5ce4-0f8077efb790
+    uuid: 5cd14e06-93ec-5fa2-ee40-5c0bd34a65b9
   ert:
     experiment:
       id: 00000000-0000-0000-0000-000000000000
@@ -105,7 +105,7 @@ masterdata:
       uuid: ad214d85-8a1d-19da-e053-c918a4889310
 source: fmu
 tracklog:
-- datetime: '2025-05-23T10:18:22.337018Z'
+- datetime: '2025-06-11T07:38:11.089580Z'
   event: created
   sysinfo:
     fmu-dataio:

--- a/schemas/0.12.0/fmu_results.json
+++ b/schemas/0.12.0/fmu_results.json
@@ -1171,6 +1171,7 @@
       "description": "The simulation mode ert was run in. These definitions come from\n`ert.mode_definitions`.",
       "enum": [
         "ensemble_experiment",
+        "ensemble_information_filter",
         "ensemble_smoother",
         "es_mda",
         "evaluate_ensemble",

--- a/src/fmu/dataio/_models/fmu_results/enums.py
+++ b/src/fmu/dataio/_models/fmu_results/enums.py
@@ -76,6 +76,7 @@ class ErtSimulationMode(str, Enum):
     `ert.mode_definitions`."""
 
     ensemble_experiment = "ensemble_experiment"
+    ensemble_information_filter = "ensemble_information_filter"
     ensemble_smoother = "ensemble_smoother"
     es_mda = "es_mda"
     evaluate_ensemble = "evaluate_ensemble"

--- a/src/fmu/dataio/_models/fmu_results/fmu_results.py
+++ b/src/fmu/dataio/_models/fmu_results/fmu_results.py
@@ -50,6 +50,8 @@ class FmuResultsSchema(SchemaBase):
     VERSION_CHANGELOG: str = """
     #### 0.12.0
 
+    - `fmu.ert.simulation_mode` now supports `ensemble_information_filter`
+
     #### 0.11.0
 
     - `data.standard_result` now supports `FluidContactSurfaceStandardResult`

